lib/saluki-components/src/sources/dogstatsd/mod.rs

```diff
@@ -257,9 +259,16 @@ impl SourceBuilder for DogStatsDConfiguration {
     }
 
     fn outputs(&self) -> &[OutputDefinition] {
-        static OUTPUTS: &[OutputDefinition] = &[OutputDefinition::default_output(DataType::Metric)];
-
-        OUTPUTS
+        static OUTPUTS: Lazy<Vec<OutputDefinition>> = Lazy::new(|| {
```

Comment:
`named_output` is not const so I used `once_cell` per the editor's suggestion in order to update `OUTPUTS`

Reply:
Actually, thinking about it... I'd also be fine if we bumped to Rust 1.80.0 (just change the value in `rust-toolchain.toml`) and used `std::sync::LazyLock`, to keep the slightly cleaner closure-in-variable-declaration pattern that we're using `Lazy` to achieve.